.github/workflows/basic-tests.yml

```
@@ -380,7 +380,7 @@ jobs:
           --task-sdk-version 1.0.0rc1 --sync-branch v3-1-test --answer yes --dry-run
       - name: "Check Airflow release process command"
         run: >
-          breeze release-management start-release --version 3.1.0
+          breeze release-management start-release --version 3.1.1
```

```suggestion
          breeze release-management start-release --version 3.1.5
```
We can use the latest too. What do you think?
[3.1.5rc1/](https://dist.apache.org/repos/dist/dev/airflow/3.1.5rc1/)

---

Yep.